build-tools/packages/build-tools/src/fluidBuild/tasks/task.ts

```diff
@@ -65,25 +65,47 @@ export abstract class Task {
 		// This function should only be called by task with task names
 		assert.notStrictEqual(this.taskName, undefined);
 		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-		this.dependentTasks = this.node.getDependentTasks(this, this.taskName!, pendingInitDep);
+		this.dependentTasks = this.node.getDependsOnTasks(this, this.taskName!, pendingInitDep);
```

Comment:
I wonder why the earlier assertion doesn't narrow the type here so the ! isn't needed?

Reply:
Node's typing for the assert function probably doesn't have assertions.